src/components/RadioBorderedBox/index.tsx

```diff
@@ -27,6 +25,10 @@ type BorderedBoxTypes = {
   error: string | ReactNode
 }
 
+const StyledRadio = styled(Radio)`
+  margin-right: ${({ theme }) => theme.space['1']};
+`
+
```

Comment:
Use a gap on parent

Reply:
i will do it for all margin right, sorry 😅

Reply:
done